src/main/com/example4/radek/DrawMeSquareApplication.java

```diff
@@ -0,0 +1,90 @@
+package main.com.example4.radek;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class DrawMeSquareApplication {
+
+    public static void main(String[] args) {
+
+
+        int size = getEnteredValue("How big square you wanna draw ? ");
+        String symbol = getEnteredValue();
+        int variant = getEnteredValue("Test all 3 variants by typing 1,2 or 3");
+        drawMeSquare(size, symbol, variant);
+    }
+
+    static void drawMeSquare(int size, String symbol, int variant) {
+
+        switch (variant) {
+            case 1:
+                /*První varianta for cyklus a potom převod na IntStream a vykreslování pomocí forEach + lambda*/
+                for (int i = 0; i < size; i++) {
+                    final int finalI = i;
+                    IntStream.range(0, size).forEach(result -> {
+                        if (result == 0 || result == size - 1 || finalI == 0 || finalI == size - 1) {
+                            System.out.print(symbol);
+                        } else {
+                            System.out.print(" ");
+                        }
+
+                    });
+                    System.out.println();
+                }
+                break;
+            case 2:
+                /*Druhá varianta převodu na IntStream a pomocí forEach + lambda vykreslit*/
```

Comment:
psal jse, at streamy a lambdu nechas jeste stranou a ten obri IF se blbe cte ... budes to muset vyresit nebo odstran case 2

Reply:
Jen jsem zkoušel na co sám příjdu